src/components/externalLinks/ExternalLinks.component.tsx

```diff
@@ -183,23 +253,35 @@ export const AppLevelExternalLinks = ({
                     </TippyCustomized>
                 )}
             >
-                <a
-                    key={linkOption.label}
-                    href={getParsedURL(true, linkOption.value.toString(), details)}
-                    target="_blank"
-                    className="external-link-chip flex left bc-n50 h-24 br-4 cn-7 dc__no-decor dc__border"
-                    rel="noreferrer"
-                >
-                    <img
-                        className="icon-dim-16 mr-4"
-                        src={linkOption.icon}
-                        alt={linkOption.label}
-                        onError={onImageLoadError}
-                    />
-                    <span className="dc__ellipsis-right" data-testid="overview_external_link_value">
-                        {linkOption.label}
-                    </span>
-                </a>
+                <div className="dc__grid br-4 dc__border dc__align-items-center external-link-chip">
+                    <div
+                        className="flexbox dc__gap-4 px-6 py-2 dc__align-items-center"
+                        role="button"
+                        onClick={() => handleOpenModal(linkOption, externalLinkURL)}
+                    >
+                        <img
+                            className="icon-dim-16"
```

Comment:
The img with span below seems to be duplicated please create a common method/component, and can use ImageWithFallback here as well